enterprise/server/scheduling/scheduler_server/scheduler_server_test.go

```diff
@@ -31,6 +33,8 @@ import (
 	ctxpb "github.com/buildbuddy-io/buildbuddy/proto/context"
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
 	scpb "github.com/buildbuddy-io/buildbuddy/proto/scheduler"
+	flagd "github.com/open-feature/go-sdk-contrib/providers/flagd/pkg"
+	"github.com/open-feature/go-sdk/openfeature"
```

Comment:
nit: move up above the non-named imports

Reply:
done